composer.lock

```diff
@@ -419,27 +419,29 @@
         },
         {
             "name": "doctrine/instantiator",
-            "version": "1.1.0",
+            "version": "1.2.0",
```

Comment:
These changes to composer.lock were intentionally added to this PR in the PHPCS fixes commit?